src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java

```diff
@@ -33,7 +33,7 @@
     private final Config cfg;
 
     private Inventory inventory;
-    private int size;
+    private static final String CONFIG_PREFIX = "backpacks.";
```

Comment:
Also I still think this should be at the top, above this line `private final PlayerProfile profile;` with an empty line above it.